modin/tests/pandas/dataframe/test_join_sort.py

```diff
@@ -174,6 +181,15 @@ def test_join(test_data, test_data2):
         df_equals(modin_join, pandas_join)
 
 
+@pytest.mark.parametrize("how", ["left", "inner", "right"])
+def test_join_empty(how):
+    data = np.random.randint(0, 100, size=(64, 64))
+    eval_general(
+        *create_test_dfs(data),
+        lambda df: df.join(df.iloc[:0], on=1, how=how, lsuffix="_caller"),
```

Comment:
Why do we not use `on` parameter for a similar test with merge?

Reply:
Different threads of execution, for `merge` function we can omit this parameter so that a certain part of the code starts executing.